corehq/apps/accounting/models.py

```diff
@@ -985,6 +985,11 @@ def _process_upgrade(domain, upgraded_privileges, new_plan_version, web_user):
             raise SubscriptionChangeError("The upgrade was not successful.")
 
 
+class SubscriptionBaseManager(models.Manager):
+    def get_queryset(self):
+        return super(SubscriptionBaseManager, self).get_queryset().filter(is_hidden_to_ops=False)
```

Comment:
More of a question than a code review comment, but is there a reason to use filter vs exclude?